.github/actions/docker/action.yml

```diff
@@ -126,15 +122,31 @@ runs:
       run: echo BUILD_ARGS=${{inputs.BUILD_ARGS}} >> $GITHUB_ENV
       shell: bash
 
-    - name: Build and export to Docker
+    - name: Build and export to Docker without cache
       uses: docker/build-push-action@v6
-      if: inputs.TAG_ONLY == 'false'
```

Comment:
TAG_ONLY is never used as far as I can tell, in any consuming repos, so removing

Reply:
`TAG_ONLY` looks to be used in `workflows/cd-retag.yml`. We should remove references to `TAG_ONLY` there as well.